Sources/BraintreeCoreSwift/BTAPIClientSwift.swift

```diff
@@ -1,5 +1,440 @@
+    }
+
+    /// :nodoc:
+    @objc(GET:parameters:httpType:completion:)
+    public func get(_ path: String, parameters: NSDictionary? = nil, httpType: BTAPIClientHTTPTypeSwift, completion: @escaping RequestCompletion) {
+        fetchOrReturnRemoteConfiguration { [weak self] configuration, error in
+            guard let self else { return }
+
+            if let error {
+                completion(nil, nil, error)
+                return
+            }
+
+            self.httpForType(httpType)?.get(path, parameters: parameters, completion: completion)
+        }
+    }
+
+    /// :nodoc:
+    @objc(POST:parameters:httpType:completion:)
+    public func post(_ path: String, parameters: NSDictionary? = nil, httpType: BTAPIClientHTTPTypeSwift, completion: @escaping RequestCompletion) {
+        fetchOrReturnRemoteConfiguration { [weak self] configuration, error in
+            guard let self else { return }
+
+            if let error {
+                completion(nil, nil, error)
+                return
+            }
+
+            self.httpForType(httpType)?.post(path, parameters: parameters, completion: completion)
+        }
+    }
+
+    ///  :nodoc: This method is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
+    public func sendAnalyticsEvent(_ eventName: String) {
+        // TODO: Implement when BTAnalyticsService is converted to Swift
+    }
+
+    // MARK: Analytics Internal Methods
+    // TODO: Implement when BTAnalyticsService is converted to Swift
+
+    /// By default, the `BTAnalyticsService` instance is static/shared so that only one queue of events exists.
+    /// The "singleton" is managed here because the analytics service depends on `BTAPIClient`.
+    /// - Returns: A `BTAnalyticsService` instance
+//    func analyticsService() -> BTAnalyticsService {
+//        // TODO: Implement when BTAnalyticsService is converted to Swift
+//    }
+
+    func queueAnalyticsEvent() {
+        // TODO: Implement when BTAnalyticsService is converted to Swift
+    }
+
+    func metadataParameters() -> [String: Any] {
+        // TODO: Implement when BTAnalyticsService is converted to Swift
+        return [:]
+    }
+
+    func graphQLMetadata() -> [String: Any] {
+        // TODO: Implement when BTAnalyticsService is converted to Swift
+        return [:]
+    }
+
+    func metaParametersWith(parameters: [String: Any], forHTTPType: BTAPIClientHTTPTypeSwift) -> [String: Any] {
+        // TODO: Implement when BTAnalyticsService is converted to Swift
+        return [:]
+    }
+
+    // MARK: - Internal Static Methods
+
+    static func baseURLFromTokenizationKey(_ tokenizationKey: String) -> URL? {
+        let pattern: String = "([a-zA-Z0-9]+)_[a-zA-Z0-9]+_([a-zA-Z0-9_]+)"
+        guard let regularExpression = try? NSRegularExpression(pattern: pattern) else { return nil }
```

Comment:
Looked into using Swift Regex, but it seems to only be available with iOS 15+ so it seems that we may need to continue to use NSRegularExpression for the time being.